examples/advanced_tensorflow/README.md

```diff
@@ -44,14 +43,16 @@ If you don't see any errors you're good to go!
 
 ## Run Federated Learning with TensorFlow/Keras and Flower
 
-The included `run.sh` will call a script to generate certificates (which will be used by server and clients), start the Flower server (using `server.py`), sleep for 2 seconds to ensure the the server is up, and then start 10 Flower clients (using `client.py`). You can simply start everything in a terminal as follows:
+The included `run.sh` will call a script to generate certificates (which will be used by server and clients), start the Flower server (using `server.py`), sleep for 2 seconds to ensure the the server is up, and then start 2 Flower clients (using `client.py`). You can simply start everything in a terminal as follows:
```

Comment:
Can we go back to ten clients here? One part of the advanced example is to show some of the things one would do when running with more than two clients. The performance of the system is largely determined by how many clients are selected during a single round, not so much by how many clients there are in total.

Reply:
@danieljanes It indeed works with 10 clients, I changed it back!